src/components/screens/all-software-screen.tsx

```diff
@@ -35,36 +36,38 @@ import SearchIcon from "@mui/icons-material/Search";
 import useCreateSoftware from "src/hooks/use-create-software";
 import AddSoftwareModal from "../software-registry/AddSoftwareModal";
 
-const statusOptions = [
-  { value: "ALL", label: "Show all" },
-  { value: "PENDING", label: "Pending" },
-  { value: "UNDER_REVIEW", label: "Under review" },
-  { value: "ACCEPTED", label: "Accepted" },
-  { value: "DEPRECATED", label: "Deprecated" },
-  { value: "DECLINED", label: "Declined" },
-];
+
 
 /**
  * All software screen component
  */
-const AllSoftwareScreen: React.FC = () => {
+const AllSoftwareScreen = () => {
   const { softwareApi } = useLambdasApi();
-  const [applications, setApplications] = useState<SoftwareRegistry[]>([]);
-  const [loading, setLoading] = useState<boolean>(true);
+  const [software, setApplications] = useState<SoftwareRegistry[]>([]);
+  const [loading, setLoading] = useState(true);
   const auth = useAtomValue(authAtom);
   const loggedUserId = auth?.token?.sub ?? "";
   const adminMode = UserRoleUtils.adminMode();
-  const [selectedStatus, setSelectedStatus] = useState<string>("ALL");
+  const [selectedStatus, setSelectedStatus] = useState<SoftwareStatusFilterOptions>("ALL");
```

Comment:
Can you not reference the enum/ allStatusValues directly here when initializing the state rather than just passing a string?